src/components/m-accordion-item/_template.js

```diff
@@ -1,6 +1,8 @@
 import html from 'nanohtml';
 import classnames from 'classnames';
 
+import '../a-icon';
```

Comment:
Why `a-icon` here?

Reply:
Because the template below outputs `<axa-icon>`